pyanaconda/ui/tui/spokes/software.py

```diff
@@ -90,6 +91,9 @@ def _initialize(self):
         # Apply the initial selection
         self._apply()
 
+        # report that the software spoke has been initialized
+        self.initialize_done()
```

Comment:
Could be a problem, that there is a new thread created in `self._apply()` for the software selection check?